pkgs/development/python-modules/duckdb-engine/default.nix

```diff
@@ -35,18 +36,22 @@ buildPythonPackage rec {
   ];
 
   preCheck = ''
-    export HOME="$(mktemp -d)"
+    HOME="$(mktemp -d)"
```

Comment:
This should be kept exported otherwise we might surprisingly lose it when forking.